test/v1/services/AmendPensionsReliefsServiceSpec.scala

```diff
@@ -83,11 +92,13 @@ class AmendPensionsReliefsServiceSpec extends UnitSpec {
         }
 
       val input = Seq(
+        "INVALID_CORRELATIONID"     -> InternalError,
```

Comment:
Again, I think we can split this up into val errors and val extraTysErrors and concatenate them together